packages/utilities/components/max.ts

```diff
@@ -0,0 +1,17 @@
+function calculate(a, b) {
+  return b > a ? b : a;
+}
+
+function max(a?: number | Date | string, b?: number | Date | string) {
+  if (arguments.length === 0) {
```

Comment:
Is there a special reason why you allow this?

Reply:
Basically, because that is how it is solved in ramda.